test/BuyTshirt.spec.ts

```diff
@@ -22,30 +22,42 @@ describe('Buy a t-shirt', () => {
   const signInStepPage: SignInStepPage = new SignInStepPage();
   const summaryStepPage: SummaryStepPage = new SummaryStepPage();
 
-  beforeAll(async () => {
-    await browser.get(webpage);
-
-    await menuContentPage.goToTShirtMenu();
-    await productListPage.goToTShirtDetail();
-    await productDetailPage.goToTShirtModal();
-    await productAddedModalPage.goToSummary();
-    await summaryStepPage.goToSignIn();
-
-    await signInStepPage.sendMail(email);
-    await signInStepPage.sendPassword(password);
-    await signInStepPage.signIn();
-
-    await addressStepPage.goToShipping();
-
-    await shippingStepPage.conditionsAgree();
-    await shippingStepPage.goToPayment();
-
-    await paymentStepPage.goToBank();
-    await bankPaymentPage.goToOrderResume();
-  });
-
-  it('then should be bought a t-shirt', async () => {
-    await expect(orderResumePage.order())
-   .toBe('Your order on My Store is complete.');
+  describe('When the webpage gets opened', () => {
+    beforeAll(async() => {
+      await browser.get(webpage);
+    });
+
+    describe('and I start the t-shirt buying process', () => {
+      beforeAll(async() => {
+        await menuContentPage.goToTShirtMenu();
+        await productListPage.goToTShirtDetail();
+        await productDetailPage.goToTShirtModal();
+        await productAddedModalPage.goToSummary();
+        await summaryStepPage.goToSignIn();
+      });
+
+      describe('and I sign In', () => {
```

Comment:
es necesario tener este describe?

Reply:
En el taller está un describe para el loggeo en la aplicación mmm, debería quitarlo?

Reply:
si lo piensas en BDD la idea es que sea muy especifico, el sign in se puede evaluar para dejarlo, lo dejo a tu criterio, para ello te puedes preguntar, si pongo este nombre en el paso me dice algo que valga la pena? es decir, si lees los otros steps sabes que es lo que se quiere hacer en el flujo?